views/view.html

```diff
@@ -0,0 +1,40 @@
+{% if data.piece %}
+  {% set openGraphUrl = data.piece._url %}
+  {% if data.piece.openGraphTitle %}
+    {% set openGraphTitle = data.piece.openGraphTitle %}
+  {% endif %}
+  {% if data.piece.openGraphDescription %}
+    {% set openGraphDescription = data.piece.openGraphDescription %}
+  {% endif %}
+  {% if data.piece.openGraphType %}
+    {% set openGraphType = data.piece.openGraphType %}
+  {% endif %}
+  {% if data.piece.openGraphImage %}
+    {% if apos.images.first(data.piece.openGraphImage) %}
```

Comment:
If there isn't one on the piece should it fall back to the page's?

Reply:
Would it have to fall back to the settings of the index page then? Is that confusing?

Reply:
That's what I was thinking. I'm honestly not sure if it'd be better to get whatever photo from the piece or a chosen photo from the index page. Probably the former, so no change needed.